freqtrade/rpc/telegram.py

```diff
@@ -126,14 +126,20 @@ def cleanup(self) -> None:
     def send_msg(self, msg: Dict[str, Any]) -> None:
         """ Send a message to telegram channel """
 
+        '🔵'.encode('ascii', 'namereplace')
+        '🚀'.encode('ascii', 'namereplace')
+        '✳'.encode('ascii', 'namereplace')
+        '❌'.encode('ascii', 'namereplace')
+        '⚠'.encode('ascii', 'namereplace')
+
```

Comment:
Now not to scare you off - but i think these lines are "debug" lines - to get to the `\N{xxx}` expressions ... ;) so please remove them (here and in the test file)

and don't worry about commit history - if necessary we can squash the commit so it'll result as one commit in the final branch.

Reply:
Not going to lie, I thought these were necessary, I will remove them :P